### DISCUSSION TITLE
Set a transaction fee limitation?

### DISCUSSION THREAD

**Discussion by bingoko** [dogecoin/dogecoin#2720] [General]
In Sept, Bitfinex just spent $23.7 million in fees to make a single Ethereum transaction by mistake. Luckily, the miner returns all the fees (https://www.theblockcrypto.com/post/118753/bitfinex-just-spent-23-7-million-in-fees-to-make-one-ethereum-transaction). However, this mistake actually can be prevented. This mistake is very similar to the "Fat Finger Error" (https://www.investopedia.com/terms/f/fat-finger-error.asp).

A simple way to prevent it is just setting an up limit check, i.e., if the order price is ±20% of the current price. Therefore, I believe it should be good to set such restrictions on the transaction fees: [0.01, 0.1] Doge. 

It is also beneficial to the community, it speeds up the adoption process of the lowing transaction fee.

Any thought?

**Comment by rnicoll** [comment]
It's in Dogecoin already: https://github.com/dogecoin/dogecoin/blob/master/src/validation.h#L61

Obviously we can't control what others run, so if they override that it's on them, but the default implementation certainly has a safety limit in.
